docs/underscore.util.strings.js.md

```diff
@@ -95,4 +117,4 @@ _.toQuery({ forms: { perfect: "circle", imperfect: "square" } });
 // => "forms%5Bperfect%5D=circle&forms%5Bimperfect%5D=square"
 ```
 
---------------------------------------------------------------------------------
\ No newline at end of file
+--------------------------------------------------------------------------------
```

Comment:
My editor automatically adds a newline to the end of the file if one doesn't exist